source/mongodump.txt

```diff
@@ -1098,3 +1107,22 @@ connect to a MongoDB Atlas cluster:
 .. code-block:: none
 
    mongodump 'mongodb+srv://cluster0.example.com/testdb?authSource=$external&authMechanism=MONGODB-AWS'  <other options>
+
+.. _mongodump-auth-dump:
+
+Authenticating to a Specific Database With ``mongodump``
```

Comment:
I think we should say "against" or "with", not "to". The docs on `authSource` don't actually ever use a preposition that we could copy, unfortunately.

Reply:
Updated to with